### DISCUSSION TITLE
MINERS VS ACCOUNTS

### DISCUSSION THREAD

**Discussion by Cowlcc** [revoxhere/duino-coin#1484] [Ideas]
Hi All
I have started mining for the first time ever. i have programmed a few projects in the past.

i noticed the my account is set to 50 max miners. lop left of the dash board..

my question is can this be exceeded. im thinking of around 100 miners. this would be under 1 ip address...

**Comment by Grantrocks** [comment]
Yes this can be exceeded for a cost. You have to go to the shop in the web wallet and purchase some more miner slots.
